Project/Packages/packages-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "com.unity.barracuda": {
-      "version": "1.3.3-preview",
+      "version": "1.4.0-preview",
```

Comment:
This should be addressed in another PR.

Reply:
Yeah, accidentally added. Will remove (or merge latest).